GENERATORS/SIMULATIONS/FEEDBACK/FEEDBACK.py

```diff
@@ -20,4 +26,9 @@ def FEEDBACK(
     if result:
         return result
     else:
-        return default
+        return (
```

Comment:
Why was this changed?

Reply:
This was implemented by @smahmed776 to get around the fact that in some topologies the feedback node didn't direct the execution order to the correct next node

Reply:
That's interesting, I guess because of the None return value? Does it still happen after you added the initial value to the feedback nodes?
Either way,  I didn't catch this before saying to change the return annotation back to `Optional[DataContainer]`, in this case you were right before and it should be `Any` instead

Reply:
Changed back to Any return type :ok_hand: